src/instructlab/config/init.py

```diff
@@ -190,10 +189,6 @@ def prompt_user_to_choose_vendors(
             )
         )
         click.echo("[0] NO SYSTEM PROFILE")
-    elif system_profile_selection == 0:
```

Comment:
What was the reasoning for removing this message?  It seems unrelated to the change.

Reply:
Apologies.  That is an error on my part.  I intended to remove the dead 'else:' code in `prompt_user_to_choose_profile()`